google-cloud-clients/google-cloud-logging/src/test/java/com/google/cloud/logging/SinkInfoTest.java

```diff
@@ -33,7 +33,7 @@
   private static final String NAME = "name";
   private static final String FILTER =
       "logName=projects/my-projectid/logs/syslog AND severity>=ERROR";
-  private static final VersionFormat VERSION = VersionFormat.V1;
+  private static final VersionFormat VERSION = VersionFormat.V2;
```

Comment:
What happens if the user explicitly sets `VersionFormat.V2`?

Reply:
If it is different from current implementation then please handle this situation.

Reply:
it was already used static i have just changed version V2 from V1 and static is fine because same object will share with test cases which are using VERSION

Reply:
Ok